diff --git a/project.godot b/project.godot
--- a/project.godot
+++ b/project.godot
@@ -25,6 +25,5 @@
 [autoload]

 G="*res://game/app/globals.gd"
-DiscordRPCLoader="*res://addons/discord-rpc-gd/nodes/discord_autoload.gd"

 [debug]
@@ -44,7 +43,7 @@
 [editor_plugins]

-enabled=PackedStringArray("res://addons/discord-rpc-gd/plugin.cfg", "res://addons/gut/plugin.cfg", "res://addons/indent_settings/plugin.cfg")
+enabled=PackedStringArray("res://addons/gut/plugin.cfg", "res://addons/indent_settings/plugin.cfg")

 [filesystem]

 import/blender/enabled=false
